src/pages/Product/Body/Comment/Comment.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+// import './Body.scss';
+// import imgA from '../../../assets/images/backimg1.jpg';
+import imgB from '../../../../assets/images/backimg2.jpg';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp } from '@fortawesome/free-solid-svg-icons';
+import { faComment } from '@fortawesome/free-solid-svg-icons';
```

Comment:
```suggestion
import { faThumbsUp, faComment } from '@fortawesome/free-solid-svg-icons';
```
이런식으로 더 간단하게 표기할 수도 있습니다!

Reply:
수정했습니다. 감사합니다.